flow/docs/grpc-api.md

```
@@ -0,0 +1,2396 @@
+
+<a name="v1-ValidateComponentsResponse"></a>
+
+### ValidateComponentsResponse
+
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| diffs | [ComponentDiff](#v1-ComponentDiff) | repeated |  |
+| total_diffs | [int32](#int32) |  |  |
+| missing_count | [int32](#int32) |  | Summary counts
```

markdownlint-cli2 (0.22.1)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)